build/pulsar_ci_tool.sh

```diff
@@ -154,6 +154,59 @@ function ci_move_test_reports() {
   )
 }
 
+function ci_check_ready_to_test() {
+  if [[ -z "$GITHUB_EVENT_PATH" ]]; then
+    >&2 echo "GITHUB_EVENT_PATH isn't set"
+    return 1
```

Comment:
`exit 1`?

Reply:
return works here too. I replaced the other occurence of `exit 1` with `return 1` to be consistent.